controller/workitem.go

```diff
@@ -141,17 +171,14 @@ func (c *WorkitemController) Update(ctx *app.UpdateWorkitemContext) error {
 	if err != nil {
 		jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError(err.Error()))
 	}
-	authorized, err := authz.Authorize(ctx, ctx.ID)
+	authorized, err := authorizeWorkitemEditor(ctx, c.db, spaceID, *ctx.Payload.Data.ID, currentUserIdentityID.String())
```

Comment:
Move to https://github.com/alexeykazakov/almighty-core/blob/6fde0adb79440b71e0960ead594b5c7054c0c917/controller/workitem.go#L186 or so, and reuse the loaded WI instead of loading it twice?